rock/stream_aligner_test_helpers.rb

```diff
@@ -0,0 +1,61 @@
+# frozen-string-literal: true
+
+# no-doc
+module Syskit
+    module Aggregator
+        # Test helpers for stream aligner
+        module TestHelpers
+            # write `sample` to `port` and waits until status reader `latest_time` equals
+            # sample time
+            def stream_aligner_write(
+                task, port, sample, sample_time_field: "timestamp", &block
+            )
+                reader = stream_aligner_status_reader(task)
+                writer = syskit_create_writer port
+                expect_execution { writer.write(sample) }
+                    .to do
+                        have_one_new_sample(reader).matching do |t|
+                            t.latest_time == sample[sample_time_field]
+                        end
+                        # let the caller add more predicates
+                        instance_eval(&block) if block
+                    end
+            end
+
+            def stream_aligner_status_reader(task)
```

Comment:
Please document the other methods

For internal methods, mark as `@api private`, e.g.

``` ruby
# @api private
#
# Create a port reader to receive the stream aligner status from `task`
#
# These readers are cached and disconnected on teardown
```

Reply:
Can you please double check them? I added a longer description trying to make it clearer.